release-1.8/scalability_validation_report.md

```diff
@@ -40,4 +42,5 @@ Validated large cluster correctness under the following configuration:
 - <job-name, run#> of the validating run (to know other specific details from the logs): https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-scale-correctness/13
 
 Misc:
-- Seeing a performance regression in density test on 5k-node with api call latency (most notably 'patch node-status') - [#51899](https://github.com/kubernetes/kubernetes/issues/51899)
+- Audit-logging showing a performance issue on GCE ([#51899](https://github.com/kubernetes/kubernetes/issues/51899)) - we disabled it for our performance tests
```

Comment:
not sure I'd call this out here since audit logging is not even enabled in production settings

Reply:
Do you suggest removing the point or adding clarity about it?

Reply:
I'd remove the audit point. `delete pods` is the only known issue in 1.8.0 with production settings as noted above.

Reply:
Done.